Gateway/Http/Client/AbstractPayment.php

```diff
@@ -25,6 +23,18 @@ class Payment implements ClientInterface
      */
     const PROCESS_STATUS_FAILED = 'failed';
 
+    /**
+     *
+     * @var string
+     */
+    const CHARGE_ID = 'charge_id';
```

Comment:
@jacstn I see this `CHARGE_ID` constant has been declared several times in multiple places (here, the request builder class, the observer class). Should we reduce the amount of this duplication?

Maybe we don't need this constant here and instead, just user from the builder class?

Reply:
this will be the subject of another PR, I will think about what single place should be owning that const.